### PR TITLE
Start blogpost for Istio's 4th birthday.

### PR DESCRIPTION
**Pull request by aijamalnk**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
The default test failure is known, and I should have a fix in soon.

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+In February 2021, we celebrated the first IstioCon! This community-led event was an opportunity
+for users and developers to share [many examples](https://www.youtube.com/playlist?list=PL7wB27eZmdffS-g_xh7X-b0echc_XZMKV)
+of how they use Istio in production and lessons
+learned from it. IstioCon was a great opportunity to feature more than 25 end-user companies,
+like Salesforce, T-Mobile, and Airbnb, among others, to feature maintainers from across the Istio
+ecosystem, and to share the [Istio project roadmap](https://www.youtube.com/watch?v=WmjTeN-jtdY).
+
+This inaugural community conference was a major success, with more than 4,000 registrants from 80
+countries that participated in this conference. The program was conducted on US and Asia time zones,
```
Comment (ericvn) [inline content/en/blog/2021/istio-4th-birthday/index.md:46]: Doesn't sound correct. Maybe: s/that participated in this conference/participating

Comment (ericvn) [inline content/en/blog/2021/istio-4th-birthday/index.md:46]: c/on US/during US

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+and on English and Chinese languages to accommodate big user communities in various continents.
```
Comment (ericvn) [inline content/en/blog/2021/istio-4th-birthday/index.md:47]: s/and on/and in

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Contributions are key to the long life of an open source project. This is why, on its 4th birthday,
+we want to hear about your contributions to the Istio project. To participate in this campaign,
+share on Twitter a contribution you made to the project and why it matters, using the hashtag
+#IstioTurns4 and #IstioBirthday. You can submit posts from Monday, May 24th at 9 am PT, until
+Friday, May 28th, at 12pm PT, to enter a chance to win some Istio swag.
+
+The other way of participating in this campaign is by joining the Isito community meetup, which
+will take place on Thursday, May 27th at 10 am PT. At this event, we will have Pratima Nambiar
+present about contributions that stem from using Isito in production at Salesforce. Join the event,
```
Comment (ericvn) [inline content/en/blog/2021/istio-4th-birthday/index.md:62]: s/present about/discuss.  Not sure how to change this.

**Review by ericvn — COMMENTED** [review]
Some quick comments.

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+description: "Celebrating Istio’s 4th birthday."
+publishdate: 2021-05-24
+attribution: "Maria Cruz (Google), Aizhamal Nurmamat kyzy (Google)"
+keywords: [community,birthday,celebration]
+---
+# Celebrating Istio’s 4th birthday
+
+Four years ago today, the Istio project was born to the open source world. To celebrate this anniversary,
+we are hosting a week-long birthday celebration  that focuses on contributions to the Istio project that
+stem from using Istio in production. Read on to learn how to participate in this celebration  and enter a
```
[the remark this answers is not in the document]
Comment (aijamalnk) [reply content/en/blog/2021/istio-4th-birthday/index.md]: done

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+## A year of important developments for Istio
+
+Over the last 12 months, the Istio project has been very focused on the [day-0
+& day-1](https://dzone.com/articles/defining-day-2-operations) experience for
+users by actively listening to our users through UX surveys and GitHub issues.
+
+- We [simplified the control plane architecture](https://istio.io/latest/blog/2020/istiod/) and
+   made Istio easier to install, configure and upgrade.
+- We provided clarity and process to our feature status and promotion of features and APIs.
+- We simplified debugging experience with various istioctl commands.
```
[the remark this answers is not in the document]
Comment (aijamalnk) [reply content/en/blog/2021/istio-4th-birthday/index.md]: done

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+- We expanded the mesh to services running in [VMs](https://istio.io/latest/news/releases/1.9.x/announcing-1.9/#virtual-machine-integration-beta)
+    and [multiclusters](https://istio.io/latest/docs/setup/install/multicluster/).
+- We made [statefulset easier](https://istio.io/latest/blog/2021/statefulsets-made-easier/) to use in Istio 1.10 with zero-configuration.
```
[the remark this answers is not in the document]
Comment (aijamalnk) [reply content/en/blog/2021/istio-4th-birthday/index.md]: done

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+- We made various performance improvements to Istio control plane and data plane via [discovery selectors](https://istio.io/latest/blog/2021/discovery-selectors/),
```
[the remark this answers is not in the document]
Comment (aijamalnk) [reply content/en/blog/2021/istio-4th-birthday/index.md]: done

File: content/en/blog/2021/istio-4th-birthday/index.md (hunk 1 above)
Comment (aijamalnk) [reply content/en/blog/2021/istio-4th-birthday/index.md]: fixed

File: content/en/blog/2021/istio-4th-birthday/index.md (hunk 3 above)
Comment (aijamalnk) [reply content/en/blog/2021/istio-4th-birthday/index.md]: fixed. WDYT?

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+Over the last 12 months, the Istio project has been very focused on the [day-0
+& day-1](https://dzone.com/articles/defining-day-2-operations) experience for
+users by actively listening to our users through UX surveys and GitHub issues.
+
+- We [simplified the control plane architecture](https://istio.io/latest/blog/2020/istiod/) and
+   made Istio easier to install, configure and upgrade.
+- We provided clarity and process to our feature status and promotion of features and APIs.
+- We simplified the debugging experience with various istioctl commands.
+- We expanded the mesh to services running in [VMs](https://istio.io/latest/news/releases/1.9.x/announcing-1.9/#virtual-machine-integration-beta)
```
Comment (ericvn) [inline content/en/blog/2021/istio-4th-birthday/index.md:28]: To satisfy the 7 lint link errors, all absolute links should be of the form [/news/releases/1.9.x/announcing-1.9/#virtual-machine-integration-beta]. See https://istio.io/latest/docs/releases/contribute/shortcodes/#add-links-to-other-pages.

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+### How to participate in Istio’s 4th Birthday celebration
+
+Contributions are key to the long life of an open source project. This is why, on its 4th birthday,
+we want to hear about your contributions to the Istio project. To participate in this campaign,
+share on Twitter a contribution you made to the project and why it matters, using the hashtag #IstioTurns4
+and #IstioBirthday. You can submit posts from Monday, May 24th at 9 am PT, until
+Friday, May 28th, at 12pm PT, to enter a chance to win some Istio swag.
+
+The other way of participating in this campaign is by joining the Isito community meetup, which
```
Comment (ericvn) [inline content/en/blog/2021/istio-4th-birthday/index.md:60]: The linter also is pointing out 7 spelling errors. Istio is misspelled on this line, and the second line after this.

The best way to see the misspelled words (the linter test output isn't highlighted) is to run the linter locally and the misspellings are highlighted in red.

Comment (ericvn) [reply content/en/blog/2021/istio-4th-birthday/index.md:60]: I did run the linter locally, and the highlighted words are `multiclusters`, #IstioTurns4, `#IstioBirthday`, `12pm`, 'meetup`, ad 2x`Isito`.

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+description: "Celebrating Istio’s 4th birthday."
+publishdate: 2021-05-24
+attribution: "Maria Cruz (Google), Aizhamal Nurmamat kyzy (Google)"
+keywords: [community,birthday,celebration]
+---
+
+## Celebrating Istio’s 4th birthday
+
+Four years ago today, the Istio project was born to the open source world. To celebrate this anniversary,
+we are hosting a week-long birthday celebration  that focuses on contributions to the Istio project that
```
Comment (ericvn) [inline content/en/blog/2021/istio-4th-birthday/index.md:12]: Still have two spaces after celebration.

File: content/en/blog/2021/istio-4th-birthday/index.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+Learn more about the [impact](https://events.istio.io/istiocon-2021/slides/IstioCon2021-Report.pdf)
+of IstioCon and find the [presentations](https://events.istio.io/istiocon-2021/sessions/) on
+the conference website.
+
+### How to participate in Istio’s 4th Birthday celebration
+
+Contributions are key to the long life of an open source project. This is why, on its 4th birthday,
+we want to hear about your contributions to the Istio project. To participate in this campaign,
+share on Twitter a contribution you made to the project and why it matters, using the hashtag #IstioTurns4
+and #IstioBirthday. You can submit posts from Monday, May 24th at 9 am PT, until
```
Comment (ericvn) [inline content/en/blog/2021/istio-4th-birthday/index.md:57]: Should we make PT Pacific as well?

**Comment by ericvn** [comment]
A diff output that did get through lint: 
[blogdiff.txt](https://github.com/istio/istio.io/files/6533148/blogdiff.txt)

**Comment by linsun** [comment]
LGTM - I've provided feedback/edited this in google doc earlier.

**Review by therealmitchconnors — APPROVED** [review]

**Comment by aijamalnk** [comment]
@ericvn PTAL, and thank you for blogdiff.txt :)

**Review by ericvn — APPROVED** [review]

**Pull request merged**